receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -30,6 +39,13 @@ metrics:
       monotonic: false
       value_type: int
       aggregation_temporality: cumulative
+  kafka.broker.log_retention_hours:
+    enabled: true
+    description: log retention time (hours) of a broker
+    unit: "h"
```

Comment:
I believe from [semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units) that the time unit of durations need to be in seconds.

Reply:
Thanks, I have changed the units to follow standard norms, Also made appropriate time conversions from logic side.